lib/pwsh.rb

```diff
@@ -54,7 +54,7 @@ def self.instance(cmd, args, options = {})
       if manager.nil? || !manager.alive?
         # ignore any errors trying to tear down this unusable instance
         begin
-          manager&.exit
+          manager.exit if defined?(manager)
```

Comment:
This should really be `manager.exit unless manager.nil?`

`defined?(manager)` will still be true it `manager = nil`